bin/smack.py

```diff
@@ -379,6 +469,7 @@ def smackdOutput(corralOutput):
     traces = []
     for traceLine in corralOutput.splitlines(True):
       traceMatch = re.match('(' + FILENAME + ')\((\d+),(\d+)\): Trace: Thread=(\d+)  (\((.*)\))?$', traceLine)
+      traceAssumeMatch = re.match('(' + FILENAME + ')\((\d+),(\d+)\): Trace: Thread=(\d+)  (\((\W*\w+\W*=\W*\w+\W*)\))$', traceLine)
```

Comment:
This extracts a variable symbol and its assumed value, and makes it available from the match groups.  This is then used to add assume statements to the witness file to tell the witness checker what values were assumed for variables at various points.